src/main/java/com/google/sps/CloudVisionAPI.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+/**
+ * Interface used for Cloud Vision API related functionality.
+ */
+public interface CloudVisionAPI {
+  protected BatchAnnotateImagesResponse querier(List<AnnotateImageRequest> requests)
```

Comment:
As we discussed yesterday, methods normally should have names indicating actions. In the case here, it should be "query". However, even "query" is not a good name here because the "CloudVisionAPI" is not an API to query something. It's an API to extract/detect texts from an image. So a better name could be "detectTexts" or "extractTexts".

Reply:
Yes yes, I was thinking about how it would sound as a whole, like, "CloudVisionAPI.querier" /query would make sense together. Also this interface can be used for different Cloud Vision API' s queries too and not just text. Will it be fine if I give it a very general name like "query" or "detect" maybe.